.github/workflows/php.yml

```
@@ -0,0 +1,52 @@
+name: PHP Static Analysis & Tests
+
+on:
+  push:
+    branches: [ develop, 'issue/592' ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  static-analysis:
+    name: Static Analysis
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: ['ubuntu-latest']
```

Do you think, it is a good idea to use `ubuntu-latest`? What, if they release `Ubuntu 22.04` and your required dependencies needs to be installed differently there or... Are not available at all?

That's the reason, why I would pin the (current) latest version:
```suggestion
        operating-system: ['ubuntu-20.04']
```

---

I just copy/pasted from an existing workflow of mine 🤷‍♂️

---

p.s. why-not-both.gif 🤔

---

Both is also fine. At least one pinned version. :D